components/automate-ui/src/app/components/chart-progress-bar/chart-progress-bar.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="chart-progress-bar" [ngStyle]="styles.container">
+    <div class="threshold point-1" [ngStyle]="styles.threshold.point1"></div>
+    <div class="threshold point-2" [ngStyle]="styles.threshold.point2"></div>
+    
+    <div id="progress" class="progress" [ngStyle]="styles.progress"></div>
+    <chef-tooltip for="progress">{{ tooltips.value }}</chef-tooltip>
+
+    <div id="bar" class="bar" [ngStyle]="styles.bar"></div>
+    <chef-tooltip for="bar">{{ tooltips.max }}</chef-tooltip>
```

Comment:
We shouldn't render tooltips if the consumer doesn't provide `valueTooltip` and `maxTooltip` inputs:

![Screen Shot 2020-04-27 at 7 02 59 PM](https://user-images.githubusercontent.com/479121/80435632-4b7d0300-88ba-11ea-9bb5-aae3b8fbccef.png)

Reply:
@scottopherson resolved 👍🏻